src/core/prompts/tools/insert-code-block.ts

```diff
@@ -0,0 +1,35 @@
+import { ToolArgs } from "./types"
+
+export function getInsertCodeBlockDescription(args: ToolArgs): string {
+	return `## insert_code_block
+Description: Inserts code blocks at specific line positions in a file. This is the primary tool for adding new code (functions/methods/classes, imports, attributes etc.) as it allows for precise insertions without overwriting existing content. The tool uses an efficient line-based insertion system that maintains file integrity and proper ordering of multiple insertions. Beware to use the proper indentation. This tool is the preferred way to add new code to files.
+Parameters:
+- path: (required) The path of the file to insert code into (relative to the current working directory ${args.cwd.toPosix()})
+- operations: (required) A JSON array of insertion operations. Each operation is an object with:
+    * start_line: (required) The line number where the code block should be inserted
```

Comment:
I wonder if it would be clearer to the model if we said this:
```suggestion
    * start_line: (required) The line number where the code block should be inserted. The content currently at that line will end up below the inserted content.
```